### ISSUE TITLE
HttpxInterceptor has no disable but a deactivate method

### ISSUE THREAD

**Issue by vincbo** [h2non/pook#125]
Interceptors are expected to have the activate/disable method pair. The newly added HttpxInterceptor has a misnamed deactivate instead of disable.

This prevents disabling httpx interceptor and can make all subsequent calls to httpx to fail.

**Comment by vincbo** [comment]
Here is the diff of the fix:

```
diff --git a/src/pook/interceptors/_httpx.py b/src/pook/interceptors/_httpx.py
index 1f11058..39589ba 100644
--- a/src/pook/interceptors/_httpx.py
+++ b/src/pook/interceptors/_httpx.py
@@ -42,7 +42,7 @@ class HttpxInterceptor(BaseInterceptor):
     def activate(self):
         [self._patch(path) for path in PATCHES]

-    def deactivate(self):
+    def disable(self):
         [patch.stop() for patch in self.patchers]
```

**Comment by sarayourfriend** [comment]
Thanks for the diff, @vincbo. I'll have a PR up with tests in a few hours once I have time, with a patch release pushed as soon as I can.

**Comment by sarayourfriend** [comment]
Fix published as v1.4.3

https://github.com/h2non/pook/releases/tag/v1.4.3

https://pypi.org/project/pook/1.4.3/

Thanks again @vincbo
